### ISSUE TITLE
BWN Network

### ISSUE THREAD

**Issue by dudqls1994** [AlexeyAB/yolo2_light#77]

**Comment by AlexeyAB** [comment]
Yes, but yolov2_light only for detection: https://github.com/AlexeyAB/yolo2_light

> How to use BIT1-XNOR-inference - only for custom models (you should train it by yourself):
> 
> You should base your cfg-file on yolov3-spp_xnor_obj.cfg and train it by using this repository as usual https://github.com/AlexeyAB/darknet by using pre-trained file darknet53_448_xnor.conv.74
> Then use it for Detection-test or for getting Accuracy (mAP):
> ./darknet detector test data/obj.names yolov3-spp_xnor_obj.cfg data/yolov3-spp_xnor_obj_5000.weights -thresh 0.15 dog.jpg
> ./darknet detector map data/obj.data yolov3-spp_xnor_obj.cfg data/yolov3-spp_xnor_obj_5000.weights -thresh 0.15
